tests/new_integration_tests/test_repos.py

```diff
@@ -513,7 +513,7 @@ def test_clone_all_repos_flat(
             f"repos clone -a {TEMPLATE_REPOS_ARG} "
             f"--base-url {platform_url} "
             "--directory-layout "
-            f"{_repobee.command.repos.DirectoryLayout.FLAT}",
+            f"{fileutil.DirectoryLayout.FLAT}",
```

Comment:
This was indeed a bit weird. I think I initially placed `DirectoryLayout` in `_repobee.command.repos`, and then moved it to a separate module and forgot to change some imports.